src/marine/emcice/emcice2ioda.py

```diff
@@ -72,9 +78,12 @@ def _read(self):
                         type=str, required=True)
     parser.add_argument('-d', '--date',
                         help="base date", type=str, required=True)
```

Comment:
add metavar="YYYYMMDDHH", to explicitly make sure the user provides that.
I have been bitten in the past.